### DISCUSSION TITLE
Proxifyre and Windows firewall. How do add rules for allow traffic ?

### DISCUSSION THREAD

**Discussion by MyWhitePages** [wiresock/proxifyre#47] [Q&A]
Hello. Thank you for good soft. I have a little issue. My windows firewall allow traffic only trust exe files.  I try add Outbound rules Proxifyre exe and Proxifyre Service but  firewall not allow. Where I make mistake ? Thank you 

![image](https://github.com/user-attachments/assets/05fd4f45-f580-43f6-9156-408cced28e55)

**Comment by wiresock** [comment]
You should allow inbound connections to ProxiFyre as well. The redirection module translates outgoing packets into incoming ones. The transparent proxy within ProxiFyre accepts these redirected connections (which appear to the firewall as if they are coming from external hosts) and then forwards them through the configured SOCKS proxy.
